doc/index.md

```diff
@@ -336,15 +356,41 @@ da.interactive(loc='left') \
 .quantile(q=w_quantile, dim='lon') \
 .hvplot(ylabel='Air Temperature [K]', width=500)
 ```
-
 ```{image} ./_static/home/interactive_hvplot.gif
 ---
 alt: interactive pipeline with an hvplot output
 align: center
 ---
 ```
+:::
+
+:::{tab-item} `.pipe`
```

Comment:
I don't think this example should be on the landing page of hvPlot, which I plan to keep simple and to the point.

Reply:
Should we ask for a second opinion and close it there?

Reply:
If the example is there it is actually possible to go to the home page and see the major features in action. I can take the home page up at a conference, in a workshop or when working with a colleague and say: This is what you get. And there is a working example they can copy and continue from.

Reply:
`.interactive` is already documented on the landing page. We can't possibly put there everything you can do with `.interactive`. But for sure we'll add more examples in the docs.